.bazelrc

```diff
@@ -78,7 +78,7 @@ common --experimental_repo_remote_exec
 try-import %workspace%/.bazelrc.user
 
 # Compile database generation config.
-build:compdb --features=-layering_check
+build:compdb --features=-layering_check --java_runtime_version=remotejdk_11
```

Comment:
What's the purpose of this change?

Reply:
Compiling tests requires JRE/JDK. I'm not sure why but I think that it stems from coverage collection for lcov being in java. Because `compdb` (compilation database) indexes all targets it fails without having the java runtime defined. Same is true for coverage. Essentially, `bazel test` fails without it.